app/src/main/java/org/pragyan/dalal18/fragment/StockExchangeFragment.kt

```diff
@@ -203,4 +231,22 @@ class StockExchangeFragment : Fragment() {
         super.onPause()
         LocalBroadcastManager.getInstance(context!!).unregisterReceiver(refreshStockPricesReceiver)
     }
+    private fun getDeviceMeasurement( ) :Pair<Int,Int>{
+         var displayMetrics = DisplayMetrics()
+        activity!!.windowManager.defaultDisplay.getMetrics(displayMetrics)
+
+        val  width = displayMetrics.widthPixels
+        val height = displayMetrics.heightPixels
+        return Pair(width,height)
+    }
+    private fun setStockButtonMeasurement(){
+        val (width,height) = getDeviceMeasurement()
+Log.d(width.toString(),height.toString())
+        //set all these measurements relative to my device so will be set accordingly to any device
+        stockDecrementButton.height=height/62
```

Comment:
Why is it 31 and 62? Also why do you require height and width?

Reply:
Tried to make size relative to my device but now i have fixed this problem